hack/components/docker-utils.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```

Comment:
Could you add a comment with usage explaining what parameters are taken by the script?